### PR TITLE
Update telemetry.md

### PR DESCRIPTION
**Pull request by justinegeffen**
- Updating telemetry content for https://github.com/mattermost/mattermost-plugin-incident-response/pull/359. 
- Ideally mirror the structure of the Mattermost telemetry docs for consistency.

### CODE REVIEW

File: docs/telemetry.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ Every event we track is accompanied with metadata that help us identify each eve
 - `event originalTimestamp`: Timestamp indicating when the event actually happened. It always equals `eventTimeStamp`.
 - `type`: Type of the event. It always contains the string `track`.
```
Comment (cpoile) [inline docs/telemetry.md:15]: this needs a bit of an update -- `type` is three types, and then `action` are those in the #359 PR notes

File: docs/telemetry.md [hunk 2]
```diff
@@ -12,19 +12,56 @@ Every event we track is accompanied with metadata that help us identify each eve
...
-| Playbook deleted | Any user clicks **Delete** in the **Actions** menu next to a playbook name in the playbook configuration page and confirms. | `PlaybookID`: Unique identifier of the playbook <br>`TeamID`: Unique identifier of the team where the playbook was located.</br><br>`NumChecklists`: Number of checklists this playbook had immediately prior to deletion.</br><br>`TotalChecklistItems`: Number of checklist items, among all checklists, this playbook had immediately prior to deletion. |
+- `type`: Type of the event. There are three event types that are tracked: `incident`, `tasks`, `playbook`.
+
+For more information about telemetry, see [Mattermost Telemetry](https://docs.mattermost.com/administration/telemetry.html).
+
+## Incident data
```
Comment (justinegeffen) [inline docs/telemetry.md:19]: @cpoile, before I continue in this vein, I wanted to get your thoughts as to whether it's more scalable than the previous table. And also whether I've understood it correctly.

**Comment by justinegeffen** [comment]
@cpoile, following up on this as I'd like to merge it today. Any feedback on the structure would be great. :)

**Comment by justinegeffen** [comment]
@cpoile, could you let me know if the proposed structure for the events/properties works?

I wasn't sure whether the event would be called ``CreateIncident`` or ``actionCreate`` (for example).

File: docs/telemetry.md [hunk 3]
```diff
@@ -1,30 +1,48 @@
...
-| Playbook deleted | Any user clicks **Delete** in the **Actions** menu next to a playbook name in the playbook configuration page and confirms. | `PlaybookID`: Unique identifier of the playbook <br>`TeamID`: Unique identifier of the team where the playbook was located.</br><br>`NumChecklists`: Number of checklists this playbook had immediately prior to deletion.</br><br>`TotalChecklistItems`: Number of checklist items, among all checklists, this playbook had immediately prior to deletion. |
+- `type`: Type of the event. There are three event types that are tracked: `incident`, `tasks`, `playbook`.
+
+For more information about telemetry, see [Mattermost Telemetry](https://docs.mattermost.com/administration/telemetry.html).
+
+## Event data
```
Comment (cpoile) [inline docs/telemetry.md:27]: I think we can delete everything below here

**Review by cpoile — CHANGES_REQUESTED** [review]
I rewrote the intro to be a bit more explanatory--please edit as you see fit (or throw it out). :)

As for the event metadata: I think this is an easier way of presenting the information. I changed the events to reflect what we're collecting. It might be easies to just accept the suggestions, and then I'll probably have to go through and do another edit because it's difficult to edit this way. 

Thanks Justine!

File: docs/telemetry.md [hunk 4]
```diff
@@ -1,30 +1,72 @@
...
+- `UserID`: Unique identifier of the server.
+- `UserActualID`: Unique identifier of the user who initiated the action.
+- `type`: Type of the event. There are three event types that are tracked: `incident`, `tasks`, `playbook`.
+
+## Data collected in `incident` events
+
+- `incidentID`: Unique identifier of the incident.
+- `IsActive`: Boolean  value indicating if the incident is active.
+- `CommanderUserID`: Unique identifier of the commander of the incident.
+- `TeamID`: Unique identifier of the team where the incident channel is created
```
Comment (amyblais) [inline docs/telemetry.md:24]:
```suggestion
- `TeamID`: Unique identifier of the team where the incident channel is created.
```

File: docs/telemetry.md [hunk 5]
```diff
@@ -1,30 +1,72 @@
...
+- `CreatedAt`: Timestamp of the incident creation.
+- `PostID`: Unique identifier of the post from which the incident was created (if relevant).
+- `NumChecklists`: Number of stages in this incident.
+- `TotalChecklistItems`: Number of tasks in this incident.
+- `ActiveStage`: A number indicating the stage of the incident (0-based).
+- `Public`: True if the incident was public, false if it was private.
```
Comment (amyblais) [inline docs/telemetry.md:30]: Some of the "true" and "false" words throughout the bullet points are formatted with an in-line code, others are not.

File: docs/telemetry.md [hunk 6]
```diff
@@ -1,30 +1,49 @@
...
+- `PostID`: Unique identifier of the post from which the incident was created (if relevant).
+- `NumChecklists`: Number of stages in this incident.
+- `TotalChecklistItems`: Number of tasks in this incident.
+- `ActiveStage`: A number indicating the stage of the incident (0-based).
+- `Public`: `true` if the incident was public, `false` if it was private.
+
+## Data collected in `tasks` events
+
+- `incidentID`: Unique identifier of the incident.
+- `NewState`: If the task was marked uncompleted. If the task was marked completed, the result will be `done`.
```
Comment (cpoile) [inline docs/telemetry.md:35]:
```suggestion
- `NewState`: `null` if the task is uncompleted, `done` if the task was marked completed.
```

**Review by cpoile — APPROVED** [review]
One more change, otherwise looks great, thanks Justine!

**Pull request merged**